conf/code-lists/office-of-exits.csv

```diff
@@ -109,7 +109,6 @@ Salford National Clearance Hub,GB000383
 Scotland Frontier,GB000067
 Scotland Inland Auth. Cee/Cor,GB000063
 Scrabster,GB000062
-Sezione Operativa Territoriale di MANFREDONIA,IT014101
```

Comment:
Just to confirm - you've deleted this office because this file should only contain UK offices?

Reply:
Correct.